src/main/kotlin/org/rust/lang/core/macros/ExpandedMacroStorage.kt

```diff
@@ -510,19 +511,26 @@ class SourceFile(
         val isExpansionFile = MacroExpansionManager.isExpansionFile(file)
         val isIndexedFile = file.isValid && (isExpansionFile || shouldIndexFile(project, file))
 
-        val stages1 = if (!isIndexedFile) {
+        val invalidateFileInfos = {
             // The file is now outside of the project, so we should not access it.
             // All infos of this file should be invalidated
```

Comment:
Let's move this comment into `if (!isIndexedFile) {` body, it's mostly related to the condition itself